software/controller/lib/debug/debug_types.h

```diff
@@ -85,13 +85,19 @@ class Handler {
 };
 
 enum class Subcommand {
-  kFlushTrace = 0x00,    // disable and flush the trace
-  kDownloadTrace = 0x01, // download the trace buffer
-  kVarInfo = 0x00,       // get variable info (name, type, help string)
-  kGetVar = 0x01,        // get variable value
-  kSetVar = 0x02,        // set variable value
-  kEepromRead = 0x00,    // read value in EEPROM
-  kEepromWrite = 0x01,   // write value in EEPROM
+  kFlushTrace = 0x00,        // disable and flush the trace buffer
```

Comment:
So, I am guessing these sub-commands are not generally applicable to all debug command types. Maybe each of the command handler classes can keep their own definitions of sub-commands in their scope?